### PR TITLE
Add properties file for meshery server

### LINKED ISSUE #962: [CI] Versioning of Meshery Server
**Current Behavior**
`mesheryctl --version` only returns the client version number, and a stubbed-out version number for meshery server. When meshery server is built, the release tag (as entered into GitHub releases) is not included as an artifact in the meshery server Docker image.

**Desired Behavior**
- include a properties.yaml in the meshery server image, which includes the release tag and the git commit sha.

**Related** 
- mesheryctl - #603
- meshery rest api - #963

### PR DESCRIPTION
**Pull request by naturalett**
Signed-off-by: Lidor Ettinger <lidor.ettinger@gmail.com>

**Description**
The meshery server image will include a properties.yaml file with a release-tag and git-sha.
When the build isn't a release then the release-tag will be the <previous_tag>-<git-sha>

This PR fixes #962 

**Description**

This PR fixes # 

**Notes to reviewers**

#### Checklist

- [] [Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin).
- [] PR title includes `[<name of area>]:<description>`

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -271,6 +271,22 @@ jobs:
       uses: actions/checkout@master
       with:
         fetch-depth: 1
+    - name: Set properties file with release tag
+      if: success() && startsWith(github.ref, 'refs/tags/')
+      run: |
+        echo git-sha=${GITHUB_SHA::8} >> properties.yml
+        echo release-tag=${GITHUB_REF/refs\/tags\//} >> properties.yml
```
Comment (leecalcote) [inline .github/workflows/ci.yml:278]: @naturalett Nice! 

Can we swap the `=` for `:`? So that it is `echo git-sha:${GITHUB_SHA::8} >> properties.yml`

Comment (naturalett) [reply .github/workflows/ci.yml]: Done

**Comment by leecalcote** [comment]
Yay!!! 🥳 🎈

**Comment by kushthedude** [comment]
Hi @naturalett, this PR has broken the `docker build & push action`. Can you please take a look into it?
https://github.com/layer5io/meshery/runs/701315774

**Pull request merged**